src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -236,6 +241,60 @@ public ObservableCollection<string> LanguagesList
             }
         }
 
+        /// <summary>
+        /// PackagePathsForInstall contains the list of all package paths where
+        /// packages can be installed.
+        /// </summary>
+        public ObservableCollection<string> PackagePathsForInstall
+        {
+            get
+            {
+                if (packagePathsForInstall == null || !packagePathsForInstall.Any())
+                {
+                    var programDataPath = new Uri(Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData) + "\\");
```

Comment:
I'm not sure - but I think using `Path` instead of URI might be better because it will be cross platform, can you check on that? (ie `\\` should be `//` on linux and mac.